manifests/quickstart-feature-examples/godel-demo-default.yaml

```diff
@@ -4,5 +4,9 @@ name: godel-demo-default
 nodes:
   - role: control-plane
     image: kindest/node:v1.21.1
+  - role: worker
```

Comment:
Do we need more workers for demo cluster?

Reply:
Yes, reservation needs two worker nodes at least, and one single worker node may cause unexpected error.